aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -237,6 +242,31 @@ module aptos_framework::coin {
         coin_store.frozen = true
     }
 
+    /// Upgrade total supply to use a parallelizable implementation if it is
+    /// available.
+    public entry fun upgrade_coin_supply<CoinType>(
+        account: &signer
+    ) acquires CoinInfo{
+        let account_addr = signer::address_of(account);
+
+        // Only coin holders can upgrade supply.
```

Comment:
Only coin creators? Holders imply if I hold APT in my wallet, I can upgrade its supply.